PowerPointLabs/PowerPointLabs/ZoomToArea.cs

```diff
@@ -1,8 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;

```

Comment:
I occasionally use it for debugging. Accidentally left it in that time (forgot to run an auto-remove usings).